cmd/release/cmd/errors.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+type VersionNotFoundError struct {
```

Comment:
Error values and error types should be prefixed with "Err" so this would be:

```go
type ErrVersionNotFound struct {
```

Reply:
I followed the same structure that are used in Go's source code, Error types being defined with `Error` suffix, like [json.MarshelerError](https://pkg.go.dev/encoding/json#MarshalerError).

Reply:
Go source code is terrible "Go code". A huge portion was auto generated from C code years and years ago and lots of other reasons. Let's stick to the general community standards and prefix with "E(e)rr".